### PR TITLE
Added some magic posh to clear run history

### PR DESCRIPTION
**Pull request by Akshay-Rohatgi**

### CODE REVIEW

File: src/release_windows.go
Comment (aaditgupta21) [inline src/release_windows.go:157]: do we still wanna do verboseEnabled here?

Comment (safinsingh) [reply src/release_windows.go:157]: It should be until #58 is merged

Comment (aaditgupta21) [reply src/release_windows.go:157]: oh your right

Comment (Akshay-Rohatgi) [reply src/release_windows.go:157]: well shanes thing hasnt been integrated into aeacus yet, i can wait for shanes pr to be merged and then change mine

**Review by Altoid0 — APPROVED** [review]

**Pull request merged**
